Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -808,6 +808,8 @@
 workflows[160.2] = ['',['PhotonJets_Pt_10_5362_HI_2023','DIGIHI2023PPRECO','RECOHI2023PPRECO','HARVESTHI2023PPRECO']]
 workflows[160.3] = ['',['ZMM_5362_HI_2023','DIGIHI2023PPRECO','RECOHI2023PPRECO','HARVESTHI2023PPRECO']]
 workflows[160.4] = ['',['ZEE_5362_HI_2023','DIGIHI2023PPRECO','RECOHI2023PPRECO','HARVESTHI2023PPRECO']]
+# Patatrack Pixel Tracks on CPU
+workflows[160.501] = ['',['HydjetQ_MinBias_5362GeV_2023_ppReco','DIGIHI2023PPRECO','RAWPRIMESIMHI18','RECOHI2023PPRECOMB_PatatrackCPU','MINIHI2023PROD']]
```

Comment:
does this include the same changes as the `.501` workflow used for pp ?

Reply:
Yes (see [here](https://github.com/cms-sw/cmssw/blob/4fac7bb211bfbbd03d35543221c44a7c3b1a8e1e/Configuration/PyReleaseValidation/python/relval_steps.py#L3016)).

Reply:
OK for me.

I was surprised to see a `.501` workflow in `relval_standard.py` (the others are in `relval_2017.py` and `relval_2026.py`).

Reply:
Right, @cms-sw/pdmv-l2 does it sound to you? Or shall I move it to `relval_2017`?